src/Core/Framework/Adapter/Doctrine/Patch/AbstractAsset.php

```diff
@@ -2,11 +2,21 @@
 
 /**
  * Doctrine breaks all FK fields due namespacing. This reverts that feature
+ * They don't want to include this fix: https://github.com/doctrine/dbal/pull/5132
```

Comment:
is is possible to mark our changes somehow? I wouldn't know what to do here, if we update this file again. in SW5 we did it like this: https://github.com/shopware5/shopware/blob/5.7/engine/Library/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php#L697-L753